dbcon/joblist/jlf_tuplejoblist.cpp

```diff
@@ -556,6 +558,47 @@ void adjustLastStep(JobStepVector& querySteps, DeliveredTableMap& deliverySteps,
     deliverySteps[CNX_VTABLE_ID] = jobInfo.annexStep;
   }
 
+  //if (orderby_threads == 99)
+  //{
+    SJSTEP acs(new TupleAnnexConstantStep(jobInfo));
+
+   if (jobInfo.constantCol == CONST_COL_EXIST)
+      dynamic_cast<TupleAnnexConstantStep*>(acs.get())->addConstant(new TupleConstantStep(jobInfo));
+
+  if (jobInfo.hasDistinct)
+      dynamic_cast<TupleAnnexConstantStep*>(acs.get())->setDistinct();
+
+  TupleAnnexStep* as = dynamic_cast<TupleAnnexStep*>(deliverySteps[CNX_VTABLE_ID].get());
+  RowGroup rg2 = as->getDeliveredRowGroup();
+
+  if (jobInfo.trace)
+    cout << "Output RowGroup 2: " << rg2.toString() << endl;
+
+  AnyDataListSPtr spdlIn(new AnyDataList());
+  RowGroupDL* dlIn;
+  if (jobInfo.orderByColVec.size() > 0)
```

Comment:
This step doesn't sort so you don't need this if.

Reply:
Removed.